apps/src/templates/teacherHomepage/StudentHomepage.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import RecentCoursesCollapsible from './RecentCoursesCollapsible';
+import CollapsibleSection from './CollapsibleSection';
+import GradientNavCard from './GradientNavCard';
+import shapes from './shapes';
+import i18n from "@cdo/locale";
+
+const StudentHomepage = React.createClass({
+  propTypes: {
+    courses: shapes.courses
+  },
+
+  componentDidMount() {
+    $('#user_hero').appendTo(this.refs.userHero).show();
```

Comment:
Worth noting that these sorts of hacks also make things like testing/storybook more difficult (which is why they should be avoided if at all possible)